src/tablecloth/.ocamlformat-ignore

```diff
@@ -0,0 +1 @@
+bs/src/compat.ml
```

Comment:
Can we move censored code to its own directory so that people don’t edit it by accident?

Reply:
It had to be vendored because of a bug in Tablecloth. We'll get rid of tablecloth in a future PR.